package.json

```diff
@@ -25,7 +26,6 @@
   },
   "peerDependencies": {
     "@sanity/base": "^2.0.9",
-    "@sanity/react-hooks": "^2.0.9",
```

Comment:
Removed because it's a duplicate. The same peerDependency is declared at line 31.